librz/analysis/arch/x86/x86_il.c

```diff
@@ -2732,14 +2847,14 @@ IL_LIFTER(popfq) {
 	return SEQ2(x86_il_set_flags(pop.val, 64), pop.eff);
 }
 
-RzILOpEffect *x86_push_helper_impl(RzILOpPure *val, unsigned int user_op_size, unsigned int bitness, const X86ILIns *ins) {
-	X86Mem stack_mem;
-	/* The correct register will automatically be chosen if we use RSP */
-	stack_mem.base = X86_REG_RSP;
-	stack_mem.disp = 0;
-	stack_mem.index = X86_REG_INVALID;
-	stack_mem.scale = 1;
-	stack_mem.segment = X86_REG_SS;
+RzILOpEffect *x86_push_helper_impl(RzILOpPure *val, unsigned int user_op_size, unsigned int bitness, const X86ILIns *ins, ut64 pc) {
+	// X86Mem stack_mem;
```

Comment:
What with this commented code?

Reply:
It's meant to be deleted. Totally missed it, thanks!